README.md

```diff
@@ -13,7 +13,7 @@ Suspected vulnerabilities can be reported on our [security page](https://github.
 An [audit of sudo-rs version 0.2.0](docs/audit/audit-report-sudo-rs.pdf) has been performed in August 2023.
 The findings from that audit are addressed in the current version.
 
-Sudo-rs currently is targeted for Linux-based operating systems only; Linux kernel 5.9
+Sudo-rs currently is targeted for Linux-based operating systems only; Linux kernel 5.11
```

Comment:
This is confusing. I think because this PR adds a fallback mechanism, that drops the kernel requirement (IIRC, that was 5.0, correct?). But this comment and the update to the kernel check seems to suggest it moves it into the opposite direction 😏

Reply:
Forgot to remove this. Removed the kernel version check.